tests/workflows/test_embarrassingly_parallel.py

```diff
@@ -74,11 +42,11 @@ def extract(filename: str, fs):
                             out.append((member.name, b"matplotlib" in data.lower()))
                 return out
 
-    futures = embarrassingly_parallel_client.map(extract, directories, fs=s3)
+    futures = client.map(extract, directories, fs=s3)
```

Comment:
Using `client` instead of, for example, `embarrassingly_parallel_client`, also seems like a small victory